src/patternfly/components/AppLauncher/app-launcher-menu-input.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-app-launcher__menu-input{{#if app-launcher-menu-input--modifier}} {{app-launcher-menu-input--modifier}}{{/if}}"
```

Comment:
do you think this should be `pf-c-app-launcher__menu-search-input`? to be more specific? Maybe more bulletproof in the future?

Reply:
Yep, I named this after the context switcher, but agreed and I think this element could just be named `__menu-search`